tests/test_make_charge_int.py

```diff
@@ -0,0 +1,28 @@
+import numpy
+import pytest
+from matchms import Spectrum
+from matchms.filtering import make_charge_int
+
+
+@pytest.mark.parametrize("input_charge, corrected_charge", [('+1', 1),
+                                                            ('1', 1),
+                                                            (' 1 ', 1),
+                                                            ('-2', -2),
+                                                            ([-1, "stuff"], -1),
+                                                            (['-3'], -3),
+                                                            ('n/a', 'n/a')])
```

Comment:
Looking at GNPS json
```
wget https://gnps-external.ucsd.edu/gnpslibrary/GNPS-LIBRARY.json
jq '.[].Charge' < GNPS-LIBRARY.json  |sort |uniq -c
   1008 "0"
    484 "-1"
   6310 "1"
      1 "10"
    282 "2"
     17 "3"
      1 "4"
```
Adding `"0"` would help.
```suggestion
                                                            ('0', 0),
                                                            ('n/a', 'n/a')])
```

Reply:
Thanks for checking the json file! Nice addition to the test.